.github/workflows/pythonpackage.yml

```diff
@@ -26,7 +26,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           cache: poetry
-      - run: poetry install
+      - run: |
```

Comment:
Since as you mention python 3.8 is EOL, I'm fine with dropping test support. Mind updating this to drop 3.8, which I think should eliminate the need for the extra logic here?